tests/js/server/aql/aql-index-hints.js

```diff
@@ -69,10 +77,52 @@ function indexHintSuite() {
       alternateEqualityIndex = 'skip_a';
       defaultSortingIndex = 'hash_a';
       alternateSortingIndex = 'skip_a_b';
+
+      internal.db._drop(cn2);
+      collection = internal.db._create(cn2);
+      collection.ensureIndex({type: "persistent", fields: ["value1"], name: persistentIdxName});
+      collection.ensureIndex({type: "persistent", fields: ["value2"], name: persistentIdxName2});
+      collection.ensureIndex({type: "geo", geoJson: true, fields: ["geo"], name: geoIdxName});
+      collection.ensureIndex({type: "geo", geoJson: true, fields: ["otherGeo"], name: geoIdxName2});
+      analyzers.save("geo_json", "geojson", {type: "point"}, ["frequency", "norm", "position"]);
+      let commonInvertedIndexMeta = {
+        type: "inverted", name: invertedIdxName, includeAllFields: true, fields: [
+          {"name": "geo", "analyzer": "geo_json"}
+        ]
+      };
+      collection.ensureIndex(commonInvertedIndexMeta);
+      commonInvertedIndexMeta = {
+        type: "inverted", name: invertedIdxName2, includeAllFields: true, fields: [
+          {"name": "otherGeo", "analyzer": "geo_json"}
+        ]
+      };
+      collection.ensureIndex(commonInvertedIndexMeta);
+
+      collection.insert({
+        geo: {type: "Point", coordinates: [50, 51]},
+        otherGeo: {type: "Point", coordinates: [100, 101]},
+        value1: 1,
+        value2: "abc"
+      });
+      collection.insert({
+        geo: {type: "Point", coordinates: [52, 53]},
+        otherGeo: {type: "Point", coordinates: [150, 151]},
+        value1: 2,
+        value2: "123"
+      });
     },
 
     tearDownAll: function () {
       internal.db._drop(cn);
+      internal.db._drop(cn2);
+      let hasRemoved = false;
+      do {
+        try {
+          analyzers.remove("geo_json");
+          hasRemoved = true;
+        } catch (err) {
+        }
+      } while (!hasRemoved);
```

Comment:
forced removal here could be used
e.g. 
`analyzers.remove("geo_json", true);`
So we don't need to wait until analyzer is actually "released" internally and don't need while loop in test here